script/Arcueid/Day 6/06_00_ARC06_1_1.txt

```diff
@@ -248,7 +248,7 @@ The head of the family is determined to make me talk.
 [sha:305dd6e6779f8a156cd7afc9348e865a10516f34]{
 -- Page 70, Offset 14090.
 -- ……それでも、事情を話す事はできない。
-...But even so, I can't talk to her about the situation.
+...But even so, I can't talk to her about this situation.
```

Comment:
I think `the` is correct, `this` would suggest the situation he is _currently_ in, which is explaining himself to Akiha

Reply:
I feel like Ben actually agrees with Ross? Anyhow, I prefer 'the'